modules/persistent-storage-csi-snapshots-operator.adoc

```diff
@@ -7,24 +7,25 @@
 
 The CSI Snapshot Controller Operator runs in the `openshift-cluster-storage-operator` namespace. It is installed by the Cluster Version Operator (CVO) in all clusters by default.
 
-The CSI Snapshot Controller Operator installs the CSI snapshot controller, which runs in the `csi-snapshot-controller` namespace.
+The CSI Snapshot Controller Operator installs the CSI snapshot controller, which runs in the `openshift-cluster-storage-operator` namespace.
```

Comment:
[BZ1906288](https://bugzilla.redhat.com/show_bug.cgi?id=1906288)

Reply:
I think yes.
@bertinatto could you kindly confirm from the CSI snapshot controller runs in openshift-cluster-storage-operator ns?

Reply:
@duanwei33 yes, that's correct